File: mobileapp/inventario/lib/models/productos.dart
class Productos {
  int id;
  String nombre;
  String descripcion;
  double precio;

  Productos(this.id, this.nombre, this.descripcion, this.precio);

  factory Productos.fromJson(Map<String, dynamic> json) {
    return Productos(
      json['id'],
      json['nombre'],
      json['descripcion'],
      json['precio'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'nombre': nombre,
      'descripcion': descripcion,
      'precio': precio,
    };
  }
}

File: cclapi/Models/Productos.cs
namespace cclapi.Models
{
    public class Productos(int Id, string Nombre, string Descripcion, decimal Precio)
    {
        public int Id { get; set; } = Id;
        public required string Nombre { get; set; } = Nombre;
        public string Descripcion { get; set; } = Descripcion;
        public required decimal Precio { get; set; } = Precio;
    }
}